pyresample/test/test_geometry/test_area.py

```diff
@@ -403,6 +403,35 @@ def test_cartopy_crs_latlon_bounds(self, create_test_area):
         latlong_crs = area_def.to_cartopy_crs()
         np.testing.assert_allclose(latlong_crs.bounds, [-180, 180, -90, 90])
 
+    def test_to_odc_geobox(self, stere_area, create_test_area):
```

Comment:
Can we xfail or skip if odc is missing? I think pytest has a decorator specifically for skipping if import fails.

Reply:
That's a good Idea. I will also add a test for missing `odc-geo`since that is not tested yet.

Reply:
And thanks for using the `create_test_area` fixture. It makes my work on Pyresample 2.0 much easier.

Reply:
I added a test for the case where `odc-geo` is not installed and changed the setup to include it as a requirement for testing. I thought about it and I think the test should not be skipped if `odc-geo` is not installed instead `odc-geo` should be required for testing as it is now.